hokusai/production.yml

```diff
@@ -113,3 +113,37 @@ spec:
     component: web
   sessionAffinity: None
   type: LoadBalancer
+---
+apiVersion: batch/v2alpha1
+kind: CronJob
+metadata:
+  name: kaws-update-sailthu
+spec:
+  schedule: 13 13 * * *
```

Comment:
Wondering if we need to give any thought to the time of day this runs. Is this scheduled to run at daily at 13:13 GMT, as written?

Reply:
This is ported directly from diffusion but am happy to tweak. I made the sailthru indexing job (separate PR) run at 715am every day.

Reply:
That would mean the job runs daily at 9:13 EST which sounds good 👍 
The actual time doesn't matter much for this job as long as it runs daily.